<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Salas</class>
 <widget class="QDialog" name="Salas" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Salas</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/rs.qrc" >:/png/buttons/16/irkick.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="btAdicionar" >
         <property name="text" >
          <string>Adicionar</string>
         </property>
         <property name="icon" >
          <iconset resource="../res/rs.qrc" >:/png/buttons/16/edit_add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btAlterar" >
         <property name="text" >
          <string>Alterar</string>
         </property>
         <property name="icon" >
          <iconset resource="../res/rs.qrc" >:/png/buttons/16/cell_layout.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btExcluir" >
         <property name="text" >
          <string>Excluir</string>
         </property>
         <property name="icon" >
          <iconset resource="../res/rs.qrc" >:/png/buttons/16/button_cancel.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tbSalas" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>No.</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Sala</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Andar</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
         <property name="icon" >
          <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_success.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancelar</string>
         </property>
         <property name="icon" >
          <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_fail.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../res/rs.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Salas</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Salas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
